test/contracts/samples/GlacisTokenClientSampleSource.sol

```diff
@@ -67,6 +66,29 @@ contract GlacisTokenClientSampleSource is GlacisTokenClientOwnable {
             );
     }
 
+    function sendMessageAndTokens__retriable(
```

Comment:
Essentially identical to sendMessageAndTokens__redundant

Reply:
True in a sense, maybe we should rename to plain sendMessageAndTokens since retriable is implicit